packages/ag-charts-website/project.json

```diff
@@ -7,7 +7,7 @@
     "build": {
       "outputs": ["{workspaceRoot}/dist/{projectRoot}"],
       "dependsOn": ["^build", "^docs-resolved-interfaces", "generate-examples"],
-      "executor": "@nxtensions/astro:build",
+      "command": "cd packages/ag-charts-website && astro build",
```

Comment:
Do we need to change this?

Reply:
So when I tried it with the normal nx commands, it would stall on `nx dev`. I haven't looked too much into it yet, but thought I'd do that after this gets merged.

Reply:
FYI: There is an option to specify the CWD to use, that might be a slightly better way to go?

E.g: https://github.com/ag-grid/ag-charts/blob/latest/packages/ag-charts-community/project.json#L148

Reply:
Ah I did try this before, but for some reason `--host` doesn't get passed through, so you don't get `https` in the server. Not sure how this would relate to `cwd` 🤷🏼‍♂️ 

I tried:

```
"cwd": "packages/ag-charts-website",
"command": "astro dev --port=${PORT} --host",
```

and

```
"executor": "nx:run-commands",
"options": {
  "command": "astro dev"
},
"args": "--port=${PORT} --host",
```

and also putting `--host` before `--port` (for both above), but none of it works. Any ideas @alantreadway ?

Reply:
I guess the issue with `--host` doesn't apply to `build` and `preview`, so can add it for those

Reply:
🤦🏼‍♂️ nvm, it works. It needs to be `options.cwd` as per the example @alantreadway posed